src/SDKs/ContainerRegistry/ContainerRegistry.Tests/Helpers/ContainerRegistryTestUtilities.cs

```diff
@@ -20,29 +23,25 @@ public class ContainerRegistryTestUtilities
         public const string ContainerRegistryNamespace = "Microsoft.ContainerRegistry";
         public const string ContainerRegistryResourceType = "registries";
 
+        public static string DefaultResourceGroupLocation = "westcentralus";
```

Comment:
Unrfortunately, having hard coded locatiosn cna make the tests difficult to maintain over time.  If you can't fix this now, please file an issue assigned to your team in the repo, label it as 'test debt', and reference it here.

Reply:
Fixed. This had to be hard-coded earlier due to limited service availability.